### DISCUSSION TITLE
Force 2FA on team members

### DISCUSSION THREAD

**Discussion by mejiasd3v** [coollabsio/coolify#1892] [Feature Requests]
Hi, as an admin I'd love to be able to force all team members to have 2FA enabled in order to access and edit settings. This is to prevent anyone from the team getting their email hacked and being able to change the infra.

**Comment by abernh** [comment]
+1
2FA is a must-have for such a critical service interface.

**Comment by crisog** [comment]
+1 - it would also be nice that this happens during onboarding so that it is effective immediately after launching coolify

**Comment by plukic** [comment]
+1

**Comment by Michael-Nussbaumer** [comment]
+1

**Comment by 2WeltenChris** [comment]
+1

**Comment by lvillalobos-cr** [comment]
+1

**Comment by tilmanbeyyer** [comment]
+1

**Comment by SultanDino** [comment]
+1

**Comment by 381sm016** [comment]
+1

**Comment by NrwPlay** [comment]
+1

**Comment by barryokane** [comment]
+1 : very relevant to Coolify Cloud too

**Comment by filipesousa** [comment]
+1 much needed.

**Comment by Will-Bill** [comment]
+100

**Comment by Will-Bill** [comment]
So we are all clear: An invite to a user, that doesn't enable 2FA and their password is engineered, leaves open complete access to Coolify, servers, databases and so on. This is a serious problem and I feel it should be one of the highest priorities.

**Comment by envake** [comment]
+1
